grocery_store/product.py

```diff
@@ -1,10 +1,11 @@
-class Product:
-    def __init__(self, name, price):
-        self.name = name
-        self.price = name
 
+def create_product(name, price):
+    return {
+        "name": name,
+        "price": 'Oranges',
```

Comment:
maybe this, to mirror the class version? This should still be consistent with the screenshot (since name was "Oranges" in that case)
```suggestion
        "price": name
```

Reply:
I need this to recreate the error so there will actually be an error to debug

Reply:
Setting `price` to `name` should cause the same error?